### ISSUE TITLE
Specs for results provided

### ISSUE THREAD

**Issue by topokel** [cart/amethyst-bunnymark#2]
Could you add these to the README.md?

I've posted my own results along with specs of the machine used in [this thread on lobste.rs](https://lobste.rs/s/p2ab42/amethyst_0_9_rendering_100_000_sprites_at#c_f1nwlz), which are fairly interesting along with the results of the user above me.

**Comment by cart** [comment]
Just added them!  I included specs for my godot3-bunnymark project and i guess i just got lazy here :smile:  I used the same machine for both benchmarks.

The results you linked to are definitely interesting. 

Comparing your videocard to mine: The 940m got a passmark score of 981 and the 1070 got a11203. So according to passmark (and as always, take all benchmarks with a grain of salt) my card is 11.42x faster than yours.

You got ~21,000 bunnies and i got 109,800 bunnies, so thats 5.23x faster according to bunnymark.

I guess thats on the same order :)

I'm excited to try out par_join once amethyst gets support for it (there are weird trait issues that currently prevent me from using it).  Maybe we'll see even better results if/when that lands.
